Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1118,9 +1118,20 @@ public abstract partial class BaseMeshSync : MonoBehaviour, IObservable<MeshSync
         int                                   numProps           = src.numProperties;
         Dictionary<int, IMaterialPropertyData> materialProperties = new Dictionary<int, IMaterialPropertyData>(numProps);
 
-        for (int pi = 0; pi < numProps; ++pi) {
+        for (int pi = 0; pi < numProps; ++pi)
+        {
             IMaterialPropertyData prop = src.GetProperty(pi);
-            materialProperties.Add(prop.nameID, prop);
+            // Change parallax slider value to 0.005
+            if (prop.nameID == MeshSyncConstants._Parallax)
+            {
+                CustomMaterialPropertyData parallax = new CustomMaterialPropertyData(prop);
+                parallax.floatValue = 0.005f;
+                materialProperties.Add(parallax.nameID, parallax);
```

Comment:
It seems that we are just setting the value with `0.005f` instead of clamping.
Is this intended ?

Reply:
Yes, the idea is that we don't have a good way to map the parallax values between dcc tools and blender. 
We also see some artifacts when the material is not tillable when using a strong parallax.   Since we don't have any indication if the material is tileable, we just keep the map hooked into the parallax map slot and we use the min parallax value in Unity to keep potential artifacts to a minimum.